cityofaustin/atd-moped
# Adds phase edit modal to dashboard

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/9459

apologies in advance of this is a lot of code! a lot of it is linting >.<

## Testing
**URL to test:** 
https://deploy-preview-734--atd-moped-main.netlify.app/

**Steps to test:**
go to the dashboard, hover over and click the status badge of a project in the my projects or followed projects view
a modal should pop up allowing you to add and/or edit phases
add a new phase and/or set a phase to current, close the modal
you should see the update immediately reflected on the dashboard

---
#### Ship list

https://user-images.githubusercontent.com/35410637/186276624-e331b491-495f-4241-bb92-e3cafe43186c.mov
